actions/distributionsActions.js

```diff
@@ -0,0 +1,40 @@
+import fetch from 'isomorphic-fetch';
+import { API_HOST } from '../actions';
+
+const url = '/api/distributions';
+
+const START = 'yasp/distributions/START';
+const DONE = 'yasp/distributions/DONE';
+const ERROR = 'yasp/distributions/ERROR';
+
```

Comment:
ok, also renamed the search ones.  requestRequest is a bit weird, but i guess consistency is more important.